app/socket-apis/get-points-of-ids.js

```diff
@@ -0,0 +1,44 @@
+// https://github.com/EnCiv/civil-pursuit/issues/203
+const Points = require('../models/points')
+const { ObjectId } = require('mongodb')
+
+async function getPointsOfIds(ids, callback, currentUserId) {
+  try {
+    // Fetch points by _id
+    const points = await Points.aggregate([{ $match: { _id: { $in: ids.map(id => new ObjectId(id)) } } }]).toArray()
+
+    // Fetch why points where parentId is in the ids
+    const whypoints = await Points.aggregate([
+      { $match: { parentId: { $in: ids.map(id => new ObjectId(id)) } } },
```

Comment:
parentId should be the string version of an ObjectId, not an object.
match for whyPoints should also match for userId, we only want the why points that this user has written.